src/modules/registration/components/Registration.js

```diff
@@ -1,43 +1,146 @@
-import React from 'react'
+import React, { useState } from 'react'
 import styles from '../styles/Registration.module.css'
+import axios from 'axios'
+import { LoadingSpinner } from '../../common/components/LoadingSpinner'
 
 export const Registration = () => {
+    const [firstName, setFirstName] = useState('')
+    const [lastName, setLastName] = useState('')
+    const [username, setUsername] = useState('')
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
```

Comment:
in terms of the formdata can you have a look here
https://twitter.com/DavidKPiano/status/1569361453928300545

And see if you can do an appraoch without having to set state?

Reply:
done